.codegen/service.go.tmpl

```diff
@@ -40,6 +40,16 @@ import (
   {{$element.Name}},{{end}}
 ]{{end}}{{end}}
 
+{{/* Same as printArray but with slightly different indentation. Go text templates does not
+allow defining functions in the template itself so we cannot combine the two subtemplates without
+creating a helper function in genkit */}}
+{{define "printArray2" -}}
+{{if le (len .) 5 -}}
+	[{{range $index, $element := .}}{{if ne $index 0}}, {{end}}{{$element.Name}}{{end}}]
+{{- else -}}[{{range $index, $element := .}}
+      {{$element.Name}},{{end}}
```

Comment:
It still make sense to include `{{if ne $index 0}}` condition here

Reply:
Why do we need that here?

Reply:
So that last element does not have a comma like in JSON style output

Reply:
The difference is minor. Let's skip.